ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.less

```diff
@@ -0,0 +1,131 @@
+@{customElementNamespace}layer-item {
+    display: block;
+    white-space: nowrap;
+    width: 100%;
+    overflow: hidden;
+    position: relative;
+
+    .layer-properties,
+    .layer-rearrange {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .layer-properties {
+        padding: @mediumSpacing;
+        margin-left: @minimumButtonSize;
+        width: ~'calc(100% - @{minimumButtonSize})';
+    }
+
+    .layer-rearrange-down,
+    .layer-rearrange-up {
+        z-index: 1;
+        position: absolute;
+        left: 0px;
+        opacity: 0;
```

Comment:
❓ Do we want to use `@coreTransitionTime` here or is there a more nuanced use you had in mind for it?

Reply:
What do you mean?

Reply:
I wasn't sure if `@coreTransitionTime` was intended to be used throughout the UI for things that have opacity transitions, but we may not have even wanted opacity transition with these elements anyway